config/rbac/leader_election_role.yaml

```diff
@@ -4,23 +4,15 @@ kind: Role
 metadata:
   name: leader-election-role
 rules:
-- apiGroups:
-  - ""
-  resources:
-  - configmaps
-  verbs:
-  - get
-  - list
-  - watch
-  - create
-  - update
-  - patch
-  - delete
-- apiGroups:
-  - ""
-  resources:
-  - configmaps/status
-  verbs:
-  - get
-  - update
-  - patch
+  - apiGroups:
```

Comment:
I think this should still include all the permissions required for the leader election to work, even if they are magically inherited by the cluster admin role the controller has.

Reply:
What is missing?

Reply:
found it in SC, fixed!